guardian/testapp/tests/test_shortcuts.py

```diff
@@ -1028,6 +1028,18 @@ def test_non_integer_primary_key_with_group_values(self):
             set(objects.values_list('pk', flat=True)),
             {obj_with_char_pk.pk})
 
+    def test_model_inheritance(self):
+        child_with_perm = ChildTestModel.objects.create(name="child1")
+        assign_perm('testapp.change_childtestmodel', self.user, child_with_perm)
+        child_without_perm = ChildTestModel.objects.create(name="child2")
+
+        children = get_objects_for_user(self.user, 'testapp.change_childtestmodel', ChildTestModel)
+
+        self.assertEqual(1, len(children))
+        self.assertIn(child_with_perm, children)
+        self.assertNotIn(child_without_perm, children)
+
+
```

Comment:
Hey, relatively new to the community. Might be overstepping here(my bad if I am) seems like we have an extra line here ? Not sure if there is any linting standards that are forced onto the codebase... but this kinda looks weird when compared with the rest of the code....

_Once again.. my bad if I'm overstepping._